Add Poe.com as provider

fhennerkes opened the pull request:
- mostly based on https://api.poe.com/v1/models
- pricing is only provided for per token priced models
- limits are 0 if not token based / available

fhennerkes committed:
add provider: Poe

fhennerkes committed:
update provider Poe

fhennerkes committed:
add initial models (xai, openai, anthropic, google)

fhennerkes committed:
Merge branch 'sst:dev' into dev

fhennerkes committed:
Add automated price update for Poe

fhennerkes committed:
Merge branch 'sst:dev' into poe-pricing-sync

fhennerkes committed:
False price change to test workflow

fhennerkes committed:
Update naming and pricing update script

fhennerkes committed:
Testing price update for GPT-5

Commit:
chore: sync Poe pricing

fhennerkes committed:
Merge prices from poe-pricing-sync

fhennerkes committed:
Merge branch 'sst:dev' into dev

fhennerkes committed:
add more models, update script

fhennerkes committed:
Merge branch 'sst:dev' into poe-pricing-sync

fhennerkes committed:
add context_length to update script and add missing new lines

fhennerkes committed:
Merge branch 'sst:dev' into poe-pricing-sync

fhennerkes committed:
Merge branch 'poe-pricing-sync' into dev

fhennerkes committed:
remove update script

fhennerkes committed:
Merge branch 'sst:dev' into dev

Commit:
chore: sync Poe pricing

fhennerkes committed:
Update poe/novita/qwen3-max-n.toml

fhennerkes committed:
Merge branch 'sst:dev' into poe-pricing-sync

Commit:
chore: sync Poe pricing

fhennerkes committed:
Poe: update GPT-5.1 models

Commit:
chore: sync Poe pricing

kamilio commented:
Hey Felix, could you please remove all models that don't support tool calling? We should fix them soon, but until then they don't provide a lot of value.

kamilio commented:
Hey @fwang, thanks for all the work on opencode and models.dev. 

For context, @fhennerkes is a software engineer at Quora, working on Poe monetization. We would like to commit to maintaining the prices and models published here on models.dev.

PeterDaveHello commented:
#137 has been addressing the same Poe integration since August with community discussions on data accuracy. 

It might make sense to consider consolidating the learnings from both approaches before moving forward.

Commit:
chore: sync Poe pricing

Commit:
chore: sync Poe pricing

fhennerkes committed:
poe: sonic 3, Gemini 3, qwen3

fhennerkes committed:
Merge branch 'sst:dev' into poe-pricing-sync

fhennerkes committed:
Merge branch 'sst:dev' into poe-pricing-sync

fhennerkes committed:
Merge branch 'sst:dev' into dev

fhennerkes committed:
Merge poe-pricing-sync into dev

fhennerkes committed:
Poe: remove models without tool support

fhennerkes commented:
> Hey Felix, could you please remove all models that don't support tool calling? We should fix them soon, but until then they don't provide a lot of value.

Sure, removed models that do not support tool calls yet and updated to the latest models / prices.

rekram1-node commented:
Is this one good to go yet?

kamilio commented:
Good to go!
